scripts/us_census/geojsons_low_res/download.py

```diff
@@ -90,22 +94,32 @@ def download_data(self, place='country/USA'):
         # to get the 0th element explicitly.
         assert len(geolevel[place]) == 1
         geolevel = geolevel[place][0]
+
+        level_to_fetch = geolevel
+        for i in range(level):
```

Comment:
```suggestion
```

Reply:
The need for this for loop is that if the user specifies e.g. level=2, we have to go two levels deep by composing the map LEVEL_MAP twice. Thus we go from `Country` -> `AdministrativeArea1` -> `AdministrativeArea2`, i.e. counties.